### PR TITLE
Update UG (Quick Start)

### PR DESCRIPTION
**Pull request by fusin02**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,35 +3,43 @@ layout: page
...
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+1. Open a command terminal, `cd` to change directory to the folder your TalentHub is in, and use the `java -jar talenthub.jar` command to run the application.<br>
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+   * `list person` : Lists all contacts.
```
Comment (jhqiu21) [inline docs/UserGuide.md:36]: Just one suggestion, should we change contacts to persons?

**Review by jhqiu21 — COMMENTED** [review]
lgtm overall!

File: docs/UserGuide.md (hunk 1 above)
Comment (fusin02) [reply docs/UserGuide.md]: Yes, thats a good idea. Thanks man

File: docs/images/Ui.png
Comment (wrjgold) [inline docs/images/Ui.png:1]: The image size should be the same as original else there might be rendering issues for the course dashboard

**Review by wrjgold — COMMENTED** [review]
Minor changes requested

**Review by wrjgold — APPROVED** [review]
lgtm

File: docs/UserGuide.md (hunk 1 above)
Comment (Haris-Irfan) [reply docs/UserGuide.md]: Glad to see the clarification!

**Comment by Haris-Irfan** [comment]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (seantham21) [reply docs/UserGuide.md]: Nice spot there Jing Hang, I missed that out previously!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,35 +3,43 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+TalentHub is a desktop app designed for **Celebrity Talent Managers**, to help with the **management of celebrity and relevant industrial contacts**.
 
+TalentHub is optimized for use via a **Command Line Interface** (CLI) while still having the benefits of a **Graphical User Interface** (GUI).
```
Comment (seantham21) [inline docs/UserGuide.md:8]: I like the description of our application. It is succinct yet informative!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,35 +3,43 @@ layout: page
...
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your TalentHub.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+1. Open a command terminal, `cd` to change directory to the folder your TalentHub is in, and use the `java -jar talenthub.jar` command to run the application.<br>
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```
Comment (seantham21) [inline docs/UserGuide.md:33]: Should we add an example for adding of events?

**Pull request merged**